### PR TITLE
docs: grid update

### PR DESCRIPTION
**Pull request by zero0Halo**
**Before submitting a pull request,** please make sure the following is done:

I have done **all** of the following:

- [ ] Added a top level class to all my components `'.anchor-[COMPONENT NAME]'`.
- [x] Used [conventional commits](https://www.conventionalcommits.org) for all work.
- [ ] Tested my solution on Mobile & Tablet.
- [ ] Wrote [unit tests](https://jestjs.io/docs/en/getting-started) for states and all behavior (`npm test`) and passed coverage thresholds.
- [ ] Updated snapshots for all permutations (`npm test -- -u`).
- [ ] Accounted for hover, focus, blur, visited, & error states because they are not edge cases.
- [ ] Created TODOs for known edge cases.
- [x] Documented all of my changes (inline & doc site).
- [ ] Made sure that all accessibility errors are resolved.
- [ ] Added [stories](https://storybook.js.org/docs/basics/introduction/) with knobs for all possible configurations.
- [x] De-linted and ran [prettier](https://github.com/prettier/prettier) (`npm run pretty`) on my code.
- [ ] Added name to OWNERS file for all new components
- [ ] If adding a new component, add its export to the rollup config
- [ ] package.json version is bumped (if necessary)

---------
**Outline your feature or bug-fix below**

* Updates doc site to Anchor v1.3.1

* Doc site now uses new `Grid`, `Cell`, `ResponsiveProvider` and `Adaptor` components.

* Doc site now uses `breakpoints()` utility function from `xstyled` for all media queries.

* Updated documentation for `Grid` to be about the new `Grid` and `Cell` components.

* Added documentation for `ResponsiveProvider` and `Adaptor`.

* Added octocat svg in the site's github link in the header

* Added a subsection to the *Components* side navigation, *Responsive*, and put the above mentioned components in it.

### CODE REVIEW

File: docs/src/components/HomeCard/HomeCard.component.tsx [hunk 1]
```diff
@@ -72,53 +74,44 @@ const StyledHomeCard = styled('div')<HomeCardProps>`
...
     title,
     to,
 }: HomeCardProps): React.ReactElement<any> => (
-    <StyledHomeCard disabled={disabled} breakpoint={breakpoint}>
-        <StyledLeftCol>
-            <img src={withPrefix(imgSrc)} alt={title} />
-        </StyledLeftCol>
+    <StyledHomeCard disabled={disabled}>
+        <Grid gap="0rem">
+            <StyledLeftCell width={{ xs: 3 }} middle center>
```
Comment (brandonlilly) [inline docs/src/components/HomeCard/HomeCard.component.tsx]: Couldn't this just be `width={3}`?

File: docs/src/components/Navigation/HomeTopNav/HomeTopNav.component.tsx [hunk 2]
```diff
@@ -29,11 +30,11 @@ export const HomeTopNav = ({
     return (
         <StyledHomeTopNav contentWidth={contentWidth}>
             <StyledContainer>
-                <Row>
-                    <Col offset={{ xs: 3, md: 6, lg: 11 }}>
+                <Grid gap="0rem">
```
Comment (brandonlilly) [inline docs/src/components/Navigation/HomeTopNav/HomeTopNav.component.tsx:33]: Should gap=0 be the default?

**Review by brandonlilly — APPROVED** [review]

File: docs/src/components/HomeCard/HomeCard.component.tsx (hunk 1 above)
Comment (zero0Halo) [reply docs/src/components/HomeCard/HomeCard.component.tsx]: You're right, it should be.

File: docs/src/components/Navigation/HomeTopNav/HomeTopNav.component.tsx (hunk 2 above)
Comment (zero0Halo) [reply docs/src/components/Navigation/HomeTopNav/HomeTopNav.component.tsx:33]: It seems fairly standard to have a default gap size so someone can immediately put something together and have it look "ok". If it was set to zero content would be touching and I'm not sure that's desirable. I feel like arguments could be made in either direction, but I prefer a default gap for the aforementioned reasons.

**Pull request merged**